packages/common/test/ContractUtils.js

```diff
@@ -13,10 +13,10 @@ contract("ContractUtils.js", function(accounts) {
         const erc20Contract = new web3.eth.Contract(ERC20ABI, erc20.address);
         // Allowance is not set for accounts[0] so this should fail on .call()
         const transaction = erc20Contract.methods.transferFrom(accounts[0], accounts[1], "1");
-        const config = {
+        const transactionConfig = {
           from: accounts[0]
         };
-        await ContractUtils.runTransaction({ transaction, config });
+        await ContractUtils.runTransaction({ web3, transaction, transactionConfig });
```

Comment:
prop name was changed from `config` to `transactionConfig` to be more explicit. Method now needs a `web3` instance to be passed in as well.